src/controllers/sold.land.filter.controller.ts

```diff
@@ -28,18 +33,39 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
-export const post = (req: Request, res: Response, next: NextFunction) => {
+export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
+
     logger.debugRequest(req, `POST ${config.SOLD_LAND_FILTER_PAGE}`);
+
```

Comment:
Can't see any tests for this changed functionality.

Reply:
These are now in the new PR [here](https://github.com/companieshouse/overseas-entities-web/pull/1471).